plugins/store/examples/AppSettingsManager/package.json

```diff
@@ -8,7 +8,7 @@
     "tauri": "tauri"
   },
   "devDependencies": {
-    "@tauri-apps/cli": "1.5.11",
+    "@tauri-apps/cli": "1.5.12",
```

Comment:
Todo: fix this before merge. Needs lock file update too so I can't do this on my phone rn.